frontend/src/views/components/Journey/Conclusion/component.tsx

```diff
@@ -39,7 +40,7 @@ const StaticConclusion: React.FC<StaticJourneyStep> = ({
               'is-colapsed': isColapsed,
             })}
             onClick={() => setExpansion(!isColapsed)}
-            aria-label="Colapse/expand overview panel"
+            aria-label={translations && translations['Colapse/expand overview panel']}
```

Comment:
Since we're translating here, maybe we can fix the typo: “Collapse” (double “l”).